### ISSUE TITLE
ignite ps stopped working

### ISSUE THREAD

**Issue by lukemarsden** [weaveworks/ignite#721]
I have a system where, after a reboot, `sudo ignite ps -a` has stopped working, it's just returning an empty response:

```
ubuntu@ns1001969:~$ sudo ignite ps -a
VM ID   IMAGE   KERNEL  SIZE    CPUS    MEMORY  CREATED STATUS  IPS     PORTS   NAME
ubuntu@ns1001969:~$ 
```

This is despite ignite VMs running:
```
ubuntu@ns1001969:~$ sudo docker ps
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS              PORTS               NAMES
9578c5ca547c        weaveworks/ignite:v0.7.1   "/usr/local/bin/igni…"   8 minutes ago       Up 8 minutes                            ignite-d390ef2d4380c9db
97594058ed90        weaveworks/ignite:v0.7.1   "/usr/local/bin/igni…"   10 minutes ago      Up 10 minutes                           ignite-c982c78df3f605ce
```

I can even SSH into them:
```
ubuntu@ns1001969:~$ sudo ignite ssh c982c78df3f605ce
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.43 x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage


This system has been minimized by removing packages and content that are
not required on a system that users do not log into.

To restore this content, you can run the 'unminimize' command.
Last login: Mon Oct 26 18:03:58 2020 from 10.61.0.1
root@c982c78df3f605ce:~# 
```

Where should I start to look at what went wrong?

**Comment by stealthybox** [comment]
Is /var/lib/firecracker/ still available?
Is that mounted over a network by chance?

What's the return code for the command?

Also, try `--log-level=debug` or `trace`

**Comment by stealthybox** [comment]
From Slack:
> i upgraded to master in production (sudo make install-all worked - would be great if this was documented) and i'm seeing both the disappearing-vm-trick bug and the 255-vms-ever limit issues both fixed. also i've seen no more crashes since i moved over to that other server (have yet to test moving back...)
> 2:15
even better would be great to have a new release please, even if you have to do it manually :sweat_smile:
> 2:16
would have been really nice to have release notes like ignite rm no longer taking --runtime and the change in ignite inspect output (IPAddresses is now under Network)
> 2:16
but overall things are looking more solid now which is great :slightly_smiling_face:

I'm fairly sure we can close this as fixed in the v0.8.0 release given all of the relateed bugfixes.
